### ISSUE TITLE
Create a binary image classification model

### ISSUE THREAD

**Issue by cla-cif** [cla-cif/Cherry-Powdery-Mildew-Detector#62]
Related to #17

**Comment by cla-cif** [comment]
CNN, 3 hidden layers, activation='sigmoid', loss='binary cross entropy', optimizer='RMSprop' -> 642,273 trainable parameters

**Comment by cla-cif** [comment]
Model still needs to be tuned / Overfitting!

**Comment by cla-cif** [comment]
Stable model deployed as 
1 input layer, 3 hidden layers (2 ConvLayer, 1 FullyConnected), 1 output layer. 
activation='softmax', loss='categorical cross entropy', optimizer='adagrad'
